avm/res/purview/account/tests/e2e/defaults/main.test.bicep

```diff
@@ -0,0 +1,48 @@
+targetScope = 'subscription'
+
+metadata name = 'Using only defaults'
+metadata description = 'This instance deploys the module with the minimum set of required parameters.'
+
+// ========== //
+// Parameters //
+// ========== //
+@description('Optional. The name of the resource group to deploy for testing purposes.')
+@maxLength(90)
+param resourceGroupName string = 'dep-${namePrefix}-purview-${serviceShort}-rg'
+
+@description('Optional. The location to deploy resources to.')
+param resourceLocation string = deployment().location
+
+@description('Optional. The location to deploy resources to.')
+param fixedResourceLocation string = 'eastus' // Not available all locations used for testing, so we need a fixed location
```

Comment:
I suggest using the approach we temporarily applied in the past using a variable like this

```bicep
#disable-next-line no-hardcoded-location // services used for this module are only available in major regions
var tempLocation = 'eastus'
```

instead of introducing another parameter that won't be used as a parameter. We undid the change back then in this [PR](https://github.com/Azure/bicep-registry-modules/pull/880/files).

Reply:
Same for the other tests

Reply:
I removed the location workaround completely, see the other comment.